src/hash/ConfigHash.hpp

```diff
@@ -18,6 +18,7 @@
  * including the attribution notice(s) required under Article 5 of the EUPL
  * in the end user terms of the application under an appropriate heading,
  * such notice(s) shall fulfill the requirements of that article.
+ * such notice(s) shall fulfill the requirements of that article.
```

Comment:
There is an extra which has been added here. Could you remove?

Reply:
just added a commit that should remove this duplicate